chsdi/templates/htmlpopup/hydrogeologische-karte_100.mako

```diff
@@ -0,0 +1,37 @@
+<%inherit file="base.mako"/>
+
+<%def name="preview()">${c.feature.titel or '-'}</%def>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('tt_document')}</td><td>&nbsp;</td></tr>
+<%
+     pdf_link_list = c['attributes']['pdf_list']
+     if pdf_link_list:
+        pdf_link = c['attributes']['pdf_list'].split('##')
+        nb=len(pdf_link)
+        pdf_link_new = []
+        i = 0
+        while i < nb:
+          if pdf_link[i] not in pdf_link_new:
```

Comment:
In general: what do we want to achieve with this loop? Assure we don't have duplicates in the pdf list?

If so, we could use `list(set(pdf_link))` to achieve the same in 1 line.